go/conversation/view_definition.py

```diff
@@ -583,9 +585,28 @@ def get(self, request, conversation):
         sio = StringIO()
         writer = csv.writer(sio)
         direction = request.GET.get('direction', 'inbound')
-        writer.writerows(conversation.get_aggregate_count(direction))
-        return HttpResponse(sio.getvalue(),
-            content_type='text/csv; charset=utf-8')
+        writer.writerows(self.get_aggregate_counts(conversation, direction))
+        return HttpResponse(
+            sio.getvalue(), content_type='text/csv; charset=utf-8')
+
+    def get_aggregate_counts(self, conv, direction):
```

Comment:
It's only used in one place and we have tests covering that. Separate tests covering `get_aggregate_counts()` would add extra time to the test runs without adding any real value.